mmv1/templates/terraform/examples/privateca_certificate_authority_cmek.tf.erb

```diff
@@ -1,10 +1,8 @@
 resource "google_project_service_identity" "privateca_sa" {
```

Comment:
similar comment to elsewhere -- suggest avoiding the term "CMEK" in this filename to reduce confusion